package.json

```diff
@@ -7,10 +7,11 @@
         "build": "vue-cli-service build --modern App",
         "lint": "vue-cli-service lint",
         "test:unit": "vue-cli-service test:unit",
-        "test": "yarn test:unit"
+        "test": "yarn test:unit",
+        "pr": "yarn lint && yarn build && yarn test && cd client && yarn lint && yarn build"
```

Comment:
built accounts client should be added to gitignore

Reply:
Depends. If we want to link to the build files for users to download the client, then it must be included. Otherwise the download has to be from somewhere else.

Reply:
Tracked here: https://github.com/nimiq/accounts/pull/90.